cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/service/ConnectionService.kt

```diff
@@ -14,22 +15,7 @@ import java.io.File
 
 @CordaService
 class ConnectionService(serviceHub: AppServiceHub) : SingletonSerializeAsToken() {
-    private val config = TestConfigurationsProvider.config(serviceHub.myInfo.legalIdentities.first().name.organisation)
-            ?: EmptyConfiguration
-                    .ifNot(
-                            ConfigurationProperties.fromFileOrNull(File("indyconfig", "indy.properties")),
-                            indyuser
-                    ) // file with common name if you go for file-based config
-                    .ifNot(
-                            ConfigurationProperties.fromFileOrNull(
-                                    File(
-                                            "indyconfig",
-                                            "${serviceHub.myInfo.legalIdentities.first().name.organisation}.indy.properties"
-                                    )
-                            ),
-                            indyuser
-                    )  //  file with node-specific name
-                    .ifNot(EnvironmentVariables(), indyuser) // Good for docker-compose, ansible-playbook or similar
+    private val config = ConfigHelper.getConfig(serviceHub.myInfo.legalIdentities.first().name.organisation)
 
     fun getCounterParty() = connection!!.getCounterParty()
```

Comment:
Could you use getConnection instead of "connection!!"? (related to other methods in class)

Reply:
Will fix.